.golangci.yml

```diff
@@ -53,10 +53,19 @@ linters:
     - wastedassign
     - whitespace
     - zerologlint
+    - ireturn
```

Comment:
why disable, and then configuring disabled linter ? 🤔

Reply:
oh, you right, I thought it was enable :)